tensorflow/cc/gradients/manip_grad_test.cc

```diff
@@ -0,0 +1,51 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/cc/framework/gradient_checker.h"
+#include "tensorflow/cc/ops/array_ops.h"
+#include "tensorflow/cc/ops/manip_ops.h"
```

Comment:
@siyavash can you please check below error : 
`third_party/tensorflow/cc/gradients/manip_grad_test.cc:18:10: error: module //third_party/tensorflow/cc:gradients_manip_grad_test does not depend on a module exporting 'third_party/tensorflow/cc/ops/manip_ops.h'
see http://go/cpp-features#layering_check; to fix run:
build_cleaner //third_party/tensorflow/cc:gradients_manip_grad_test
#include "third_party/tensorflow/cc/ops/manip_ops.h"`

Reply:
cc_ops (which contains manip_ops) was not included in gradients_manip_grad_test deps, but the Roll op would had been resolved from //tensorflow/core:testlib. I added cc_ops to the deps, but how can I check everything is ok now ?